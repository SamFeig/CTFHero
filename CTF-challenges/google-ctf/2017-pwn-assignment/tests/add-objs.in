a.b.c=1
a.b.d="a"
a.e=5
f=9
g=f+a
g
g=a
h=a+f
h
h=a
z.b=3
z.e.a="z"
z.e.b=4
z.d=10
y=z+a
y
