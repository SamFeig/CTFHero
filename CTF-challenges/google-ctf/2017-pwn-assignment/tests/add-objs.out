> > > > > > {
  b: {
    c: 10
    d: "9a"
  }
  e: 14
}
> > > {
  b: {
    c: 10
    d: "a9"
  }
  e: 14
}
> > > > > > > {
  b: {
    c: 4
    d: "3a"
  }
  e: {
    a: "z5"
    b: 9
  }
  d: 10
}
> Failed reading
